### PR TITLE
tmuxPlugins.tilish: unstable-2020-08-12 -> unstable-2023-09-20

### PR DESCRIPTION
**Pull request by MarkusLohmayer**
This is my first PR to this repository. I appreciate any feedback.

Maintainer: @arnarg 

## Description of changes

Changes:
https://github.com/jabirali/tmux-tilish/commits/master

Only breaking change is keybinding [M-f -> M-z](https://github.com/jabirali/tmux-tilish/commit/1c6c7eb4934f6066252abff5f96b283d13b302fb)

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -580,12 +580,12 @@ in rec {
 
   tilish = mkTmuxPlugin {
     pluginName = "tilish";
-    version = "2020-08-12";
+    version = "2023-09-20";
```
Comment (natsukium) [inline pkgs/misc/tmux-plugins/default.nix:583]:
```suggestion
    version = "2023-09-20";
```
```suggestion
    version = "unstable-2023-09-20";
```

**Review by natsukium — CHANGES_REQUESTED** [review]
Thanks for your contribution!
The changes look good. Could you please read [CONTRIBUTION.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) and change the commit message and the PR title?
It should be `tmuxPlugins.tilish: unstable-2020-08-12 -> unstable-2023-09-20`.

**Comment by MarkusLohmayer** [comment]
@natsukium Do I have so squash the two commits, or is it enough if the last commit has the desired message?

**Comment by natsukium** [comment]
Please squash them, thanks.

**Comment by MarkusLohmayer** [comment]
Done!

**Review by arnarg — APPROVED** [review]
LGTM

**Review by natsukium — APPROVED** [review]
LGTM, thanks.
Welcome to nixpkgs!🎉

**Pull request merged**